docs/core/tools/sdk-errors/netsdk1013.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "NETSDK1013: The TargetFramework value '{0}' was not recognized. It may be misspelled. If not, then the TargetFrameworkIdentifier and/or TargetFrameworkVersion properties must be specified explicitly."
```

Comment:
Titles are supposed to be limited to 65 characters, so I would include only the error text, omiting the propose solution text. -- https://review.docs.microsoft.com/en-us/help/contribute/contribute-how-to-write-good-titles?branch=master

```suggestion
title: "NETSDK1013: The TargetFramework value was not recognized."
```